samples/classes/method.jakt

```diff
@@ -1,6 +1,6 @@
 class Person {
     name: String
```

Comment:
I know it doesn't matter for this example, but for consistency with the previous and following test I'd make them both public.